src/batch-audio-transcription/CliOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using CommandLine;
+
+namespace BatchAudioTranscription
+{
+    class CliOptions
+    {
+        [Option('i', "input", Required = true, HelpText = "The folder that contains the media files to process.\r\nE.g. c:\\temp\\")]
+        public string Input { get; set; }
+
+        [Option('k', "key", Required = true, HelpText = "This is your Speech Service key found in the Keys and Endpoint blade in Azure portal")]
+        public string Key { get; set; }
+
+        [Option('g', "region", Required = true, HelpText = "This is your Speech Service Location/Region found in the Keys and Endpoint blade in Azure portal")]
```

Comment:
Use g instead of r to avoid overriding the `dotnet run -r` option which is for runtime